plugins/modules/dconf.py

```diff
@@ -143,14 +143,28 @@
 
 
 import os
+import sys
 
 from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.common.respawn import (
+    has_respawned,
+    probe_interpreters_for_module,
+    respawn_module,
+)
 from ansible.module_utils.common.text.converters import to_native
 from ansible_collections.community.general.plugins.module_utils import deps
 
+# Python 2 doesn't have ModuleNotFoundError
+try:
+    import_errors = (ImportError, ModuleNotFoundError)
+except NameError:
+    import_errors = (ImportError,)
+
+glib_module_name = 'gi.repository.GLib'
+
 try:
     from gi.repository.GLib import Variant, GError
-except ImportError:
+except import_errors:
```

Comment:
Another out of curiosity question: why use `ImportError, ModuleNotFoundError` and not just `ImportError` (as before)? Is there a case where `ModuleNotFoundError` is raised instead of `ImportError`?

Reply:
According to https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError `ModuleNotFoundError` is a subclass of `ImportError`, and thus the above can be replaced back with `except ImportError`. (That's also what all other modules in community.general and other collections do.)